packages/terra-functional-testing/tests/jest/commands/validates/index.test.js

```diff
@@ -0,0 +1,7 @@
+const { accessibility } = require('../../../../src/commands/validates');
+
+describe('index', () => {
+  it('should export the accessibility function', () => {
+    expect(accessibility).toBeDefined();
```

Comment:
Was there a test for this helper?

Reply:
There's not a jest test. This helper is being regression tested in WDIO. Setting up, mocking, and extending the internal expect used inside the function was a little tricky